.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+    paths-ignore:
+      - '**.md'
```

Comment:
```suggestion
```

The other 2 can be ignored but this is part of the required checks, can't be ignored.